CONTRIBUTING.md

```diff
@@ -8,7 +8,7 @@ Contributing to the BRJS codebase is really easy. We only have one hard rule: **
 * Check for unnecessary whitespace with `git diff --check` before committing
 * Run all the tests for the project before making the pull request
 
-When you submit a pull request please submit into `develop` rather than `master` - the BRJS uses develop for day to day work, only merging to master for releases. If you have built from a feature branch and want to submit a pull request into that feature branch feel free to submit a PR to that branch instead.
+When you submit a pull request to `master`. If you have built from a feature branch and want to submit a pull request into that feature branch feel free to submit a PR to that branch instead.
```

Comment:
You've started a sentence with "When" that you don't subsequently finish. Also, I'm not sure this entire paragraph is needed anymore.